src/pages/docs/projects/built-in-step-templates/email-notifications.md

```diff
@@ -30,19 +31,13 @@ To add your SMTP configuration navigate to **Configuration ➜ SMTP** and set th
 | From Address       | The address which all emails will be sent 'From'. | octopus@mydomain.com |
 | Credentials        | Optional SMTP login / password if your SMTP server requires authentication. | mylogin@mydomain.com / SuperSecretPa$$word |
 
-Click **SAVE AND TEST** to save the SMTP configuration and verify the values are valid:
+Click **Save and test** to save the SMTP configuration and verify the values are valid:
 
 :::figure
 ![](/docs/projects/built-in-step-templates/images/smtp-configuration.png)
 :::
 
-You will be prompted for an email address to send a test email to:
-
-:::figure
-![](/docs/projects/built-in-step-templates/images/smtp-test-email.png)
```

Comment:
This was an image of the dialog that showed up with a text input. I think it's easy enough to follow without an image.